### PR TITLE
Rename sources

### PR DESCRIPTION
**Pull request by mruwnik**
https://github.com/StampyAI/alignment-research-dataset/issues/36#

### CODE REVIEW

File: .github/workflows/fetch-dataset.yml [hunk 1]
```diff
@@ -8,32 +8,31 @@ on:
         type: choice
         options:
           - agentmodels
-          - aiimpacts.org
-          - aipulse.org
+          - aiimpacts
+          - aipulse
           - aisafety.camp
-          - alignment_newsletter
```
Comment (ccstan99) [inline .github/workflows/fetch-dataset.yml:14]: Removing alignment_newsletter since it takes ~30 minutes to scrape and shouldn't change any more.

**Review by ccstan99 — APPROVED** [review]
I've uploaded renamed files to gdrive. After you've updated data loader, feel free to push to HF. There's a zip file with previous jsonl, just in case I goofed. We can delete it if everything looks good on HF.

**Comment by mruwnik** [comment]
fixed conflicts

File: align_data/blogs/__init__.py [hunk 2]
```diff
@@ -7,25 +7,25 @@
...
     ),
-    WordpressBlog(name="vkrakovna.wordpress.com", url="https://vkrakovna.wordpress.com"),
+    WordpressBlog(name="vkrakovna_blog", url="https://vkrakovna.wordpress.com"),
     WordpressBlog(
-        name="yudkowsky.net",
+        name="yudkowsky_blog",
         url="https://yudkowsky.net",
         strip=["^\s*Download as PDF\n"]
     ),
     MediumBlog(name="deepmind.blog", url="https://deepmindsafetyresearch.medium.com/", authors=["DeepMind Safety Research"]),
```
Comment (ccstan99) [inline align_data/blogs/__init__.py:25]: We're using `deepmind_blog` with an underscore. Somehow that got undone in the last commit.

**Review by ccstan99 — APPROVED** [review]
And thanks for all the black magic to handle authors!

File: .github/workflows/fetch-dataset.yml (hunk 1 above)
Comment (mruwnik) [reply .github/workflows/fetch-dataset.yml:14]: not any more :D Now it takes ~5s

**Pull request merged**